app/models/user.rb

```diff
@@ -46,6 +46,12 @@ def app_capabilities
   has_many :appointments
   has_many :medical_histories
   has_many :prescription_drugs
+  has_many :teleconsultation_requests,
```

Comment:
Can we rename these associations to `requested_teleconsultations` and `recorded_teleconsultations`? That's more consistent with the idea that both associations return a collection of teleconsultations.